ospf6d/ospf6_area.c

```diff
@@ -454,6 +455,31 @@ void ospf6_area_show(struct vty *vty, struct ospf6_area *oa,
 		json_object_int_add(json_area, "numberOfAreaScopedLsa",
 				    oa->lsdb->count);
 
+		json_lsa_stats = json_object_new_object();
+		json_object_int_add(json_lsa_stats, "numberOfRouterLsa",
+				    oa->lsdb->stats.num_type1);
```

Comment:
I feel it would be good , if json keys are numberOfRouterLsas,  numberOfNetworkLsas, ...

Reply:
I would rather leave it in the singular form to keep it consistent with the existing JSON keys.

Reply:
@rgirada Can we consider this resolved?